api/api.proto

```diff
@@ -3374,6 +3369,10 @@ message Job {
         // NodeDrainAttachmentsJob if selected this job desribes
         // the task for removing volume attachments from a node
         NodeDrainAttachmentsJob drain_attachments = 4;
+        // CloudDriveTransferJob is selected describes the task to transfer a cloud driveset
+        // from one node to another
+        CloudDriveTransferJob clouddrive_transfer = 401;
```

Comment:
Why 401? Also you can reorder the numbers since Job message has not been released yet.

Reply:
```
  string s3_storage_class = 7;
  // Start at field number 200 to allow for expansion
  oneof credential_type {
    // Credentials for AWS/S3
    SdkAwsCredentialRequest aws_credential = 200;
    // Credentials for Azure
    SdkAzureCredentialRequest azure_credential = 201;
    // Credentials for Google
    SdkGoogleCredentialRequest google_credential = 202;
  }
```

wanted the oneof to have it's own range. e.g credential_type above.

Updated it to 400 and 401.